### DISCUSSION TITLE
`in-range` and `out-of-range` pseudo selectors

### DISCUSSION THREAD

**Discussion by hood** [tailwindlabs/tailwindcss#8114] [Ideas]
As title implies, it'd be cool to have `in-range` and `out-of-range` pseudo selectors implemented in TailwindCSS.

**Comment by adamwathan** [comment]
Hey! Already have them actually 👍🏻 

https://tailwindcss.com/docs/hover-focus-and-other-states#quick-reference

**Comment by hood** [comment]
🤦‍♂️ my bad, thanks!
